test/schema/fixtures/Choice.json

```diff
@@ -21,11 +20,10 @@
     ]
   },
   {
+    "$ref": "http://standardhealthrecord.org/schemas/shr/types.schema.json#/definitions/code",
```

Comment:
It's minor, but I think that best practice generally lists the `$schema` and `id` first.  Can we put the `$ref` below the description (where the `type` was before)?

Even though it doesn't matter to the actual processing, I'd like the fixtures to follow best practice since they also serve as our examples right now.